src/server/protocol.ts

```diff
@@ -3598,6 +3598,13 @@ export interface UserPreferences {
      * Default: `false`
      */
     readonly organizeImportsCaseFirst?: "upper" | "lower" | false;
+    /**
+     * Indicates where named type-only imports should sort. The default order ("inline") sorts imports regardless of if imports are
+     * type-only.
+     *
+     * Default: `inline`

```

Comment:
The current default is effectively "last"; for backwards compat I'm thinking we should probaby keep that?

Reply:
That makes sense to me, and is inline with the past changes to organize imports

Reply:
Of course, changing the default to inline was what I was doing when I was attempting to make this change, was very helpful in debugging.